src/component/admin/users/change-password-component.jsx

```diff
@@ -20,8 +27,17 @@ function ChangePassword({
     const [data, setData] = useState({});
     const [error, setError] = useState({});
     const [validPassword, setValidPassword] = useState(false);
+    const [showPassword, setShowPassword] = useState(false);
     const commonStyles = useCommonStyles();
 
+    const handleClickShowPassword = () => {
+        setShowPassword(!showPassword);
+    };
+
+    const handleMouseDownPassword = e => {
```

Comment:
What is the function of this?

Reply:
Just following the Material UI example here: https://mui.com/components/text-fields/#input-adornments